website/docs/d/kvstore_instances.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_kvstore_instances"
+sidebar_current: "docs-alicloud-datasource-rkv-instances"
```

Comment:
Please modify line 4 `docs-alicloud-datasource-rkv-instances` to `docs-alicloud-datasource-kvstore-instances`